webapp/channels/src/components/app_bar/__snapshots__/app_bar.test.tsx.snap

```diff
@@ -1,335 +1,79 @@
-            >
-              <div
-                className="app-bar__old-icon app-bar__icon-inner app-bar__icon-inner--centered"
-                role="button"
-                tabIndex={0}
-              >
-                fallback_component
-              </div>
-            </div>
-          </OverlayTrigger>
-        </OverlayTrigger>
-      </AppBarPluginComponent>
+          fallback_component
+        </div>
+      </div>
       <hr
-        className="app-bar__divider"
-        key="divider"
+        class="app-bar__divider"
       />
-      <AppBarBinding
-        binding={
-          Object {
-            "app_id": "com.mattermost.zendesk",
-            "label": "Create Subscription",
-          }
-        }
-        key="com.mattermost.zendesk_Create Subscription"
+      <div
+        aria-label="Create Subscription"
+        class="app-bar__icon"
+        id="app-bar-icon-com.mattermost.zendesk"
       >
-        <OverlayTrigger
-          defaultOverlayShown={false}
-          delayShow={400}
-          overlay={
-            <Tooltip
-              bsClass="tooltip"
-              id="tooltip-app-bar-icon-com.mattermost.zendesk"
-              placement="right"
-            >
-              <span>
-                Create Subscription
-              </span>
-            </Tooltip>
-          }
-          placement="left"
-          trigger={
-            Array [
-              "hover",
-              "focus",
-            ]
-          }
+        <div
+          class="app-bar__icon-inner"
         >
-          <OverlayTrigger
-            defaultOverlayShown={false}
-            delayShow={400}
-            overlay={
-              <OverlayWrapper
-                bsClass="tooltip"
-                id="tooltip-app-bar-icon-com.mattermost.zendesk"
-                intl={null}
-                placement="right"
-              >
-                <span>
-                  Create Subscription
-                </span>
-              </OverlayWrapper>
-            }
-            placement="left"
-            trigger={
-              Array [
-                "hover",
-                "focus",
-              ]
-            }
-          >
-            <div
-              aria-label="Create Subscription"
-              className="app-bar__icon"
-              id="app-bar-icon-com.mattermost.zendesk"
-              onBlur={[Function]}
-              onClick={[Function]}
-              onFocus={[Function]}
-              onMouseOut={[Function]}
-              onMouseOver={[Function]}
-            >
-              <div
-                className="app-bar__icon-inner"
-              >
-                <img />
-              </div>
-            </div>
-          </OverlayTrigger>
-        </OverlayTrigger>
-      </AppBarBinding>
+          <img />
+        </div>
+      </div>
     </div>
   </div>
-</AppBar>
+</DocumentFragment>
 `;
 
 exports[`components/app_bar/app_bar should match snapshot on mount when App Bar is disabled 1`] = `
```

Comment:
I don't see much difference between the two snapshots. Is that expected? What exactly are we trying to test with these snapshots?

Reply:
That change is introduced by switching from Enzyme's `mount` to Testing Library's `render`. I would've removed the snapshots entirely, but I don't really know what these ones are trying to test, so I left them mostly unchanged